keyboards/haverworks/theseus75/keyboard.json

```diff
@@ -0,0 +1,460 @@
+{
+    "manufacturer": "Haverworks",
+    "keyboard_name": "Theseus75",
+    "maintainer": "ebastler",
+    "bootloader": "stm32-dfu",
+    "bootmagic": {
+        "matrix": [0, 1]
+    },
+    "diode_direction": "COL2ROW",
+    "eeprom": {
+        "driver": "wear_leveling",
+        "wear_leveling": {
+            "driver": "embedded_flash"
+        }
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A8", "pin_b": "C6", "resolution": 2}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "cols": ["B11", "C14", "B6", "B5", "B4", "B3", "C11", "C10", null],
+        "rows": ["A9", "A10", "B15", "B12", "B13", "B14"]
+    },
+    "processor": "STM32G431",
+    "rgb_matrix": {
+        "animations": {
+            "alphas_mods": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_sat": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "band_val": true,
+            "breathing": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "cycle_up_down": true,
+            "dual_beacon": true,
+            "gradient_left_right": true,
+            "gradient_up_down": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "jellybean_raindrops": true,
+            "pixel_flow": true,
+            "pixel_fractal": true,
+            "pixel_rain": true,
+            "rainbow_beacon": true,
+            "rainbow_moving_chevron": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "solid_color": true
```

Comment:
```suggestion
            "raindrops": true
```

Reply:
Oh, assuming `solid_color` is enabled by default then?

Reply:
Removed in e2b4041.